frontend/packages/console-shared/src/components/spotlight/Spotlight.scss

```diff
@@ -0,0 +1,64 @@
+@keyframes expand {
+  0% {
+    outline-offset: -4px;
+    outline-width: 4px;
+    opacity: 1;
+  }
+  100% {
+    outline-offset: 21px;
+    outline-width: 12px;
+    opacity: 0;
+  }
+}
+
+@keyframes fade-in {
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+
+@keyframes fade-out {
```

Comment:
```suggestion
@keyframes co-spotlight-fade-out {
```

Reply:
done

Reply:
@christianvogt @nemesis09 Shouldn't this be `ocs-spotlight-expand` since this is in `console-shared` package?

Reply:
yes it should be. my mistake

Reply:
renamed to `ocs-spotlight-expand`